### ISSUE TITLE
GUID returned by RFC has garbled characters

### ISSUE THREAD

**Issue by MichelFernandes1997** [SAP/node-rfc#187]
I make a call to RFC and the returned data have a **GUID === L^=V�c�**
Whats happened here? Someone can help me?

**Comment by bsrdjan** [comment]
Need more info on platform, could you please post the output of:

```shell
$ node -p "require('node-rfc').environment"
```

**Comment by MichelFernandes1997** [comment]
Retuned undefined

**Comment by MichelFernandes1997** [comment]
node-rfc: "0.1.10" and node: 6.9.4. These are the versions

**Comment by bsrdjan** [comment]
You need the latest node-rfc and supported node LTS version: https://github.com/SAP/node-rfc#supported-platforms

**Comment by MichelFernandes1997** [comment]
For node-rfc it is version 2.3.1 and for node it is 12. * correct?

**Comment by MichelFernandes1997** [comment]
You can help me for install the latest node-rfc version? I execute npm install node-rfc but receive errors and my version not changed

**Comment by MichelFernandes1997** [comment]
I'll updated node-rfc and node libs but the error continue. You have idea?

**Comment by bsrdjan** [comment]
Could you please post the output of:

```shell
$ node -p "require('node-rfc').environment"
```

**Comment by PaulWieland** [comment]
Are you using docker by chance?

I found that if I had `node-rfc` installed on a volume, and I swap the container image out to a new version of nodejs but use the old version of `node-rfc` from the volume that I get garbage characters returned from RFCs.

My situation is unique, but in my case I had to make sure the volume was built from scratch so that the latest versions of my npm libraries were installed by the version of node in the docker image.
